IR-Policy.md

```diff
@@ -24,4 +24,145 @@ See the **_Applicability_** section of the [GSA IT Security Policy](http://www.g
+
+If the cloud.gov system changes in a radical way, the Program Manager adapts the incident response training to meet the needs of the new system. The Program Manager then requires Cloud Operations team members to take the training again.
+The Program Manager requires all Cloud Operations team members to take the incident response training at least once a year.
+
+See IR-2.
+
+The cloud.gov team, as directed by the Program Manager, creates test plans and exercises in accordance to NIST 800-61, and presents these to the cloud.gov Authorizing Official for their approval. 
+
+cloud.gov tests its incident response capabilities with an annual table top exercise. The test takes the form of a teleconference (GSA Google Hangout) meeting where a security specialist (such as the Program Manager, a Cloud Operations team member, or another security specialist from 18F) guides the Cloud Operations team through a role-playing exercise with a simulated potential security incident. The team takes notes throughout the test, and afterward the team discusses the test and identifies weaknesses to fix with additional training or process improvements. The team files and tracks improvements with GitHub issues.
+
+See IR-3.
+
+The cloud.gov team conducts incident response tabletop exercises to test our incident response plan, organized in coordination with Infrastructure and GSA Information Security. cloud.gov coordinates evaluates if coordination is required with other organizational elements as incident response plans change.
+
+See IR-3 (2).
+
+cloud.gov implements automated processes to detect and analyze malicious activity within the platform, as described in SI-3 and other SI controls.  If these processes detect malicious activity, they automatically report the activity to the Cloud Operations team. cloud.gov has an Incident Response Guide (https://docs.cloud.gov/ops/security-ir/) that documents the procedures that staff should take in the case of an incident, as required by the 18F and GSA Incident Response Policy.
+
+As part of that documented Incident Response process, the Cloud Operations team can use automated tools to eradicate the threat and recover to a known state.  cloud.gov uses a service-oriented architecture that allows natural containment and separation.
+
+If the system needs to be restored to a prior state, the team first reviews recent commits.  If a defect is found in the configuration, the team initiates a Github rollback and a Concourse redeploy.  This cycle will terminate the Virtual Machines and create new machines with known good configurations.  Further details are included in SI controls.
+Some cloud.gov incident response and contingency planning activities are inherently coordinated due to overlapping subject areas. The cloud.gov team plans to further improve this coordination by documenting how to coordinate incident handling activities with contingency planning activities.
```

Comment:
Actually I think we should just delete a bunch of this detail for DRY purposes, since it's already documented in the IR guide.